crypto/asn1/x_long.c

```diff
@@ -56,11 +56,30 @@ static void long_free(ASN1_VALUE **pval, const ASN1_ITEM *it)
     *(long *)pval = it->size;
 }
 
+static int num_bits_ulong(unsigned long value)
```

Comment:
For completeness sake one probably should clarify that only sizeof(long)>sizeof(BN_ULONG) is problematic. Otherwise, i.e. sizeof(long)<=sizeof(BN_ULONG), BN_num_bits_word is always usable...